src/tools/kdb/factory.hpp

```diff
@@ -110,6 +111,7 @@ class Factory
 		m_factory.insert (std::make_pair ("file", std::make_shared<Cnstancer<FileCommand>> ()));
 		m_factory.insert (std::make_pair ("sget", std::make_shared<Cnstancer<ShellGetCommand>> ()));
 		m_factory.insert (std::make_pair ("merge", std::make_shared<Cnstancer<MergeCommand>> ()));
+		m_factory.insert (std::make_pair ("newmerge", std::make_shared<Cnstancer<NewMergeCommand>> ()));
```

Comment:
Maybe call it "cmerge". In any case this is temporary, the goal is to have all current commands replaced with your merging.